source/features/jump-to-conversation-close-event.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import {observe} from 'selector-observer';
+import * as pageDetect from 'github-url-detection';
+
+import {wrap} from '../helpers/dom-utils';
+import features from '.';
+
+function init(): Deinit {
+	return observe('#partial-discussion-header :is([title="Status: Closed"], [title="Status: Merged"], [title="Status: Closed as not planned"]):not(.rgh-jump-to-conversation-close-event)', {
```

Comment:
We don't need to make this dynamic. Let's just use `select`

Reply:
The header refreshes on any title change etc. Plus there are 2, one header and one sticky header.

Reply:
ahh you're right. I think we have a listener for that? If not, that's ok, but let's move the callback out of `init`